com/main/basic/HelloWorld.java

```diff
@@ -3,6 +3,8 @@
 
 Class HelloWorld{
   public static void main(String args[]){
-    System.out.println("This is a sample class.");
+    System.out.println("Hi Ramesh, How are you?");
+    int a =10, b=20;
```

Comment:
Pls move these initialization to golabl.
```suggestion
    int a =10, b=20;
```